### ISSUE TITLE
Clarify web use

### ISSUE THREAD

**Issue by gisborne** [tekartik/sqflite#801]
Amazingly great packages. Thank you!

Just one thing to clarify: [sqflite_common](https://pub.dev/packages/sqflite_common) bears badges for all major Dart platforms, including Web.

It says it is not for use directly; instead it says:

> See [sqflite](https://pub.dev/packages/sqflite) for flutter mobile
> See [sqflite_common_ffi](https://pub.dev/packages/sqflite_common_ffi) for Desktop and unit test

The second should also say "web" if indeed that is the package to use on the web. If not, I don't know what is…

Going to [sqflite_common_ffi](https://pub.dev/packages/sqflite_common_ffi).

First, the Readme doesn't agree with the badges. It again bears badges for all the major platforms. But the Readme says:

> Currently supported on Linux, MacOS and Windows on both Flutter and Dart VM.

So, well, I'm a tad confused by that. The badges say it is suitable for mobile, but the Readme kinda sorta says it's not?

And I still don't know what to use on the web.

**Comment by alextekartik** [comment]
Yes it was not possible to control the badges before sorry, I'll need to fix that.

Currently web is not supported. You can read more about it here: https://github.com/tekartik/sqflite/issues/212

I might consider using a wasm solution in the future (maybe running in a web worker) as the sqlite3 package has now a first working solution. I also look at absurd sql but I don't see any good solution in the near future. I personally recommend using IndexedDB for the web for now.
